gulpfile.js

```diff
@@ -88,34 +88,35 @@ var paths = {
   },
 
   vendorChallenges: [
-    'public/bower_components/jshint/dist/jshint.js',
-    'public/bower_components/chai/chai.js',
-    'public/bower_components/CodeMirror/lib/codemirror.js',
-    'public/bower_components/CodeMirror/addon/comment/comment.js',
-    'public/bower_components/CodeMirror/addon/edit/closebrackets.js',
-    'public/bower_components/CodeMirror/addon/edit/matchbrackets.js',
-    'public/bower_components/CodeMirror/addon/lint/lint.js',
-    'public/bower_components/CodeMirror/addon/lint/javascript-lint.js',
-    'public/bower_components/CodeMirror/mode/javascript/javascript.js',
-    'public/bower_components/CodeMirror/mode/xml/xml.js',
-    'public/bower_components/CodeMirror/mode/css/css.js',
-    'public/bower_components/CodeMirror/mode/htmlmixed/htmlmixed.js',
+    'node_modules/jshint/dist/jshint.js',
+    'node_modules/chai/chai.js',
+    'node_modules/codemirror/lib/codemirror.js',
+    'node_modules/codemirror/addon/comment/comment.js',
+    'node_modules/codemirror/addon/edit/closebrackets.js',
+    'node_modules/codemirror/addon/edit/matchbrackets.js',
+    'node_modules/codemirror/addon/lint/lint.js',
+    'node_modules/codemirror/addon/lint/javascript-lint.js',
+    'node_modules/codemirror/mode/javascript/javascript.js',
+    'node_modules/codemirror/mode/xml/xml.js',
+    'node_modules/codemirror/mode/css/css.js',
+    'node_modules/codemirror/mode/htmlmixed/htmlmixed.js',
```

Comment:
Hm. Interesting. So this can potentially break if higher version was installed on the top level, via dependency of other module, which is alphabetically before `codemirror`. [Shrinkwrap](https://docs.npmjs.com/cli/shrinkwrap) can be used to solve this and lock down versions. I've used it and the only downside is that you have to manually update it whenever you change your package.json.